test/base/ByteBufferTest.ooc

```diff
@@ -9,6 +9,18 @@
 use base
 use unit
 
+CustomAllocator: class extends AbstractAllocator {
+	_allocCount := 0
+	init: func
+	allocate: override func (size: SizeT) -> Pointer {
+		this _allocCount += 1
+		malloc(size)
+	}
+	deallocate: override func (pointer: Pointer) {
+		memfree(pointer)
```

Comment:
Maybe have a `freeCounter` as well and make sure that we indeed call `deallocate`.